### ISSUE TITLE
Crash when disabling shaders (or reloading textures [F3+T])

### ISSUE THREAD

**Issue by vpontin** [copygirl/WearableBackpacks#31]
Using MC 1.11.2, Forge 13.20.0.2312, preview-OptiFine_1.11.2_HD_U_C1_pre and WearableBackpacks-1.11.2-2.4.1.

Crashlog: https://gist.github.com/vpontin/cc9a9d0cb70019a96c83db0f46fa8c3b

Dropping issue here due to the various `net.mcft.copy.backpacks.***` in the stacktrace.

**Comment by copygirl** [comment]
Can you test if this issue still occurs without OptiFine?

**Comment by vpontin** [comment]
Dropped the optifine jar and the crash's gone. Sorry for waste your time.
*Mental note: i really need to find an alternative, at least for better chunk loading and shaders*

**Comment by copygirl** [comment]
For the record I don't mind if you keep this issue open until it's properly fixed. Will you contact the OptiFine team for them to have a look at it?

Unfortunately I did not write the model loading code and don't understand it well at all. @asiekierka, if you don't mind, could I ask you for your expert opinion on this matter?

**Comment by vpontin** [comment]
Oh sorry, i always close issues when is determined this is related to Optifine, because several modders says "I can't cope with that".

But if your wants, i will reopen.

**Comment by asiekierka** [comment]
@vpontin Closing it doesn't mean the issue will disappear. Let's wait for OptiFine devs to weigh in first.

**Comment by copygirl** [comment]
I don't see issues as something bad. It's a reminder to myself and information to other users, maybe even developers, that a problem has been found and is present. If you were to close this, another user might be more likely to report the same issue. And I like to keep them around even if another mod is at fault, since it still affects users of mine. If you look at issue #3, it's still open even though it's marked as "wontfix".

**Comment by Seth0067** [comment]
Copygirl - great practice.  Found this useful as I noticed this issue myself. @vpontin did you ever get a response from Optifine?

**Comment by asiekierka** [comment]
The relevant issue is open on the OptiFine tracker, so this can be closed I suppose.

**Comment by vpontin** [comment]
Sorry to not report back - the fact is i ditched from optifine months ago - i became aware of the bizarre ways such closed source mod uses to "deliver performance" - not worth it since a lot of mods break ( another to be cited beyond WB is ComputerCraft; monitors simply dont work with optifine installed).

Sorry to bother asiekierka in first place, Optifine jar should be in my trash long before.

**Comment by sp614x** [comment]
Out of curiosity I checked how Wearable Backpacks works with OptiFine + shaders.

Tested Forge #2588 + OptiFine E1 for 1.11.2 + WearableBackpacks-1.11.2-2.5.0.
Loading and unloading shaders is OK, no crashes.

Also tested Forge 1.12.2 and latest OptiFine + WearableBackpacks for 1.12.2. 
Everything is OK, no crashes when enabling/disabling shaders.

ComputerCraft was fixed to work correctly with shaders: https://github.com/sp614x/optifine/issues/1237#issuecomment-377066341
It uses custom rendering that was not compatible with the shaders shadow map.
